### ISSUE TITLE
Documentation bug: U does not have a Monoid instance

### ISSUE THREAD

**Issue by raymoo** [phaazon/luminance#52]
It says at https://hackage.haskell.org/package/luminance-0.11.0.4/docs/Graphics-Luminance-Shader-Program.html#t:U that `U` is monoidal, but it doesn't seem to be the case.

**Comment by phaazon** [comment]
Yeah, I removed that because it implies several `glUniform*` underneath and discourages people from using buffers, which is wrong. I’ll update the documentation.

**Comment by phaazon** [comment]
Yeah, I removed that because it implies several `glUniform*` underneath and discourages people from using buffers, which is wrong. I’ll update the documentation.
